src/listeners/s3-sqs-listener.ts

```diff
@@ -30,14 +32,22 @@ const s3 = new AWS.S3({
 
 
 export default async function start() {
+  console.log('Starting import listener...');
   // Connection URL
-  const url = configManager.get("dbUrl");
+  const dbUrl = configManager.get("dbUrl");
   // Target bucket
   const editorBucket = configManager.get("editorS3Bucket");
   // Database Name
   const dbName = configManager.get("dbName");
-  const client = await MongoClient.connect(url);
-  const db = client.db(dbName);
+  // connect to cluster with TSL enabled 
+  const dbSSLValidate = configManager.get("dbSSLValidate");
```

Comment:
This 'might' be redundant if the URL contains the same data.

Reply:
The other PR for flux removes the cert path from the url so we just rely on `mongodb` config (as per AWS docs).